### PR TITLE
Duplicatecheck

### PR DESCRIPTION
**Pull request by davekwon555**
Duplicate check for Users and Orgs. Checks duplicate by user email_address and org name. The duplicate check only applies to new User/Org being created, not necessarily the objects inside the database. Deletion of duplicate in the database should be done manually.

### CODE REVIEW

File: features/duplicatecheck.feature [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	Then I should see "User E-mail address has already been taken"
+	And I should not see "User successfully created."
+	
+Scenario: user fills in information in new user page
+    Given I am on the Users page
+    When I follow "New User"
+    And I fill in "User Name" with "new_user"
+    And I fill in "User E-mail address" with "new_user@user.com"
+    And I press "Save"
+	Then I should see "User successfully created."
```
Comment (kyriakipapadimitriou) [inline features/duplicatecheck.feature]: SMART test! As a small edit I would recommend you change the indentation so it all looks similar and make it even more beautiful.

**Review by kyriakipapadimitriou — APPROVED** [review]
Great overall! Change the indentation of some steps of the tests. Good job

**Review by kimhoang73 — APPROVED** [review]
looks good!

**Review by csungwon — APPROVED** [review]
Everything seems great to me.

**Pull request merged**